filter/store.go

```diff
@@ -154,6 +156,7 @@ func (store *Store) makeRequest(method, url string, body io.Reader) ([]byte, err
 	if err != nil {
 		return nil, err
 	}
+	request.Header.Set("Internal-token", store.filterAPIAuthToken)
```

Comment:
The filter API uses snake case `internal_token` for the header name instead of `Internal-token`.